### ISSUE TITLE
I can make backup of a table specified in the database?

### ISSUE THREAD

**Issue by chernandezq** [ShawnMcCool/database-backup#18]
I want to separate the backup of tables, can I do it?

**Comment by ShawnMcCool** [comment]
Yes, but you need to create your own task / procedures, which is actually very easy. You can look at the source and expand it how you like. Just don't use the facade (DatabaseManager) that we provide. Bootstrap your own procedures and just use it as the facade would have.
